### PR TITLE
Add fundamental browser E2E test

### PR DESCRIPTION
**Pull request by at-wat**
Add connection establish and RTP packet count test using headless
Chromium. Run the test as GitHub Action.

ref: #42

### CODE REVIEW

File: e2e/e2e_test.go [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+
+	"github.com/pion/webrtc/v2"
+	"github.com/pion/webrtc/v2/pkg/media"
+)
+
+var silentOpusFrame = []byte{0xf8, 0xff, 0xfe} // 20ms, 8kHz, mono
+
+func TestE2E(t *testing.T) {
+	drivers := map[string]*agouti.WebDriver{
+		"Chrome": agouti.ChromeDriver(
```
Comment (at-wat) [inline e2e/e2e_test.go]: would have Firefox/GeckoDriver in future.

**Comment by Sean-Der** [comment]
:fire: this fantastic!

I have a bunch of things I want to test with this! We should be able to get WAY more media coverage now, this is really great :)

**Comment by Sean-Der** [comment]
Do you like the workflows? Should we move everything over to that?

It has been on my TODO list. The travis stuff is sometimes flakey

**Comment by at-wat** [comment]
There are still some cons in github actions at now:
- coverage report requires token; can't be triggered by PRs from forked repositories
- not fully stable yet (I met some stucks which is not able to stop from the web interface)

But, repository specific jobs like this is easy to be handled by simple add a new workflow yaml, even if the main jobs are shared between repos.

**Pull request merged**